_posts/2020-09-26-github-teams.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Translation Teams move to GitHub
+author: frosch
+---
+
+Today is a big day for translations of OpenTTD!
+Translators are now organised as teams of our [GitHub organisation](https://github.com/OpenTTD).
+
+That means:
+* You no longer need an OpenTTD-specific account to become a translator: you can use [GitHub accounts](https://github.com) for most OpenTTD things now.
+  ([bug tracker](https://github.com/OpenTTD/OpenTTD/issues), [BaNaNaS](https://bananas.openttd.org)), [translations](https://translator.openttd.org), [development](https://github.com/OpenTTD/OpenTTD), ...)
+* Becoming a translator is now super easy: you just [click a few buttons](https://github.com/OpenTTD/team/issues/new/choose) to become a member of the team.
+* There is one team for each language, which allows the translators to get to know each other, and have group conversations within the team.
+
+There is only one downside:
+* Everyone who was a translator before needs to [reapply](https://github.com/OpenTTD/team/issues/new/choose).
+
+While this is good news for all translators, this is also a great step forward for our infrastructure:
+after migrating the source code and bug tracker in 2019, this year we could migrate BaNaNaS and the WebTranslator.
+This only leaves the [Wiki](https://wiki.openttd.org) as last service that still needs an OpenTTD-specific account. (guess what task is next?)
```

Comment:
Will the Wiki remain on the current software except for the authentication part (which will soon require you to sign in with GitHub), or will the wiki itself migrate to a different software? Assuming I allow it, will my contributions migrate over (so that they're associated with my Github username instead of my old OpenTTD-specific account username)?

Also, here's a grammar fix:
```suggestion
This only leaves the [Wiki](https://wiki.openttd.org) as the last service that still needs an OpenTTD-specific account. (guess what task is next?)
```

Reply:
Whether we keep MediaWiki or use something else is undecided. We are still exploring options.
The page history and author information will most likely get migrated, but this depends on the chosen software of course.
However, if author information is kept, they will look like "james@users.openttd.org" or similar, and will not be linked to your GitHub account. Similar as in the migrated bug reports.